src/build/app/mod.rs

```diff
@@ -236,6 +236,73 @@ impl<'help> App<'help> {
         self.get_opts().filter(|a| a.get_help_heading().is_none())
     }
 
+    /// Test if the provided Argument is a global Argument
+    pub fn test_global(arg: &Arg) -> bool {
+        arg.global
+    }
+
+    /// Get a list of subcommands which contain the provided Argument
+    ///
+    /// This command will only include subcommands in its list for which the subcommands
+    /// parent also contains the Argument.
+    ///
+    /// **NOTE:** In this case only Sucommand_1 will be included
+    ///   Subcommand_1 (contains Arg)
+    ///     Subcommand_1.1 (doesn't contain Arg)
+    ///       Subcommand_1.1.1 (contains Arg)
+    ///
+    pub fn get_subcommands_containing(&self, arg: &Arg) -> Vec<&App<'help>> {
```

Comment:
Can you please add more clear docs? I have no idea what this is doing based on the docs.

Reply:
I thought that this method might be useful to other people, so I made it public. But maybe it is not all that useful.
Explaining it in the docs would lead to a long explanation involving the implementation.
Let's just make this method not public and leave the comment as is but not as a doc comment.